src/block.js

```diff
@@ -12,10 +27,12 @@ export default function Block( { currentTotal, freeShippingFrom } ) {
 		remaining > 0
 			? sprintf(
 					__(
-						'Spend only $%s more to get free US shipping!',
+						'Spend only %s%s%s more to get free shipping!',
 						'free-shipping-progress-bar'
 					),
-					remaining
+					currencyPrefix,
+					remaining,
+					currencySuffix
```

Comment:
That's not how you should constrcut the price. If you're going to that in this PR, then you would need price-format. If you're going to do this in a follow up, then revert to the original code please.

Reply:
I just removed `currencyPrefix` and `currencySuffix` and will handle the price formatting in a separate PR.